static/js/stream_popover.js

```diff
@@ -13,6 +13,30 @@ let current_topic_sidebar_elem;
 let all_messages_sidebar_elem;
 let starred_messages_sidebar_elem;
 
+function get_current_stream_sidebar_menu_items() {
+    if (!current_stream_sidebar_elem) {
+        blueslip.error("Trying to get menu items when action popover is closed.");
+        return;
+    }
+
+    const popover_data = $(current_stream_sidebar_elem).data("popover");
+    if (!popover_data) {
+        blueslip.error("Cannot find popover data for stream sidebar menu.");
+        return;
+    }
+    return $("li:not(.divider):visible a", popover_data.$tip);
+}
```

Comment:
I'd prefer not to add 6 copies of this set of 3 functions.  Can't we just make the `current_stream_sidebar_elem` parameter an argument to a single function?  The rest of the content here is the same for all of them.

(Also, I think the first conditional is useless in that we're only calling the callers to this function inside checks for whether it's already popped)